index.js

```diff
@@ -16,8 +17,10 @@ const createUnboxTransform = (queryParam) => {
   return createUnboxStream(keyBytes, zeros);
 }
 
-function ServeBlobs(sbot) {
+function ServeBlobs(sbot, config) {
   return function(req, res, next) {
+    const corsEnabled = get(config, 'serveBlobs.cors', false)
```

Comment:
put this below line 20 so its not run every request

Reply:
oops